bridges/GBAtempBridge.php

```diff
@@ -32,45 +32,62 @@ private function buildItem($uri, $title, $author, $timestamp, $thumbnail, $conte
 		return $item;
 	}
 
+	private function strEndsWith($haystack, $needle){
+		// str_ends_with is not available below PHP 8
+		$length = strlen($needle);
+		return $length > 0 ? substr($haystack, -$length) === $needle : true;
+	}
+
+	private function decodeHtmlEntities($text) {
+		$text = html_entity_decode($text);
+		$convmap = array(0x0, 0x2FFFF, 0, 0xFFFF);
+		return trim(mb_decode_numericentity($text, $convmap, 'UTF-8'));
+	}
+
 	private function cleanupPostContent($content, $site_url){
-		$content = str_replace(':arrow:', '&#x27a4;', $content);
-		$content = str_replace('href="attachments/', 'href="' . $site_url . 'attachments/', $content);
+		$content = str_replace('src="/', 'src="' . $site_url, $content);
+		$content = str_replace('href="/', 'href="' . $site_url, $content);
 		$content = stripWithDelimiters($content, '<script', '</script>');
-		return $content;
+		$content = stripWithDelimiters($content, '<svg', '</svg>');
+		$content = stripRecursiveHTMLSection($content, 'div', '<div class="reactionsBar');
+		return $this->decodeHtmlEntities($content);
+	}
+
+	private function makeAbsoluteUrl($link) {
+		if (strpos($link, '/') === 0) {
+			$link = substr($link, 1);
+		}
+		return self::URI . $link;
 	}
```

Comment:
Instead of manually working with links, maybe you can try using defaultLinkTo method?

Reply:
Indeed. Switched to the built-in/library functions and removed the custom one.